internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,467 @@
+	cmd.Flags().Bool("overwrite", false, "Overwrite existing topics with the same name.")
+	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	cmd.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	cmd.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+	cobra.CheckErr(cmd.MarkFlagRequired("file"))
+	return cmd
+}
+
+func getFlagsImport(cmd *cobra.Command) (*flagsImport, error) {
+	file, err := cmd.Flags().GetString("file")
+	if err != nil {
+		return nil, err
+	}
+	overwrite, err := cmd.Flags().GetBool("overwrite")
+	if err != nil {
+		return nil, err
+	}
+	kafkaApiKey, err := cmd.Flags().GetString("kafka-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiKey, err := cmd.Flags().GetString("schema-registry-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiSecret, err := cmd.Flags().GetString("schema-registry-api-secret")
+	if err != nil {
+		return nil, err
+	}
+
+	flags := &flagsImport{
+		file:                    file,
+		overwrite:               overwrite,
+		kafkaApiKey:             kafkaApiKey,
+		schemaRegistryApiKey:    schemaRegistryApiKey,
+		schemaRegistryApiSecret: schemaRegistryApiSecret,
+	}
+	return flags, nil
+}
+
+func (c *command) asyncapiImport(cmd *cobra.Command, args []string) error {
+	// Get flags
+	flagsImp, err := getFlagsImport(cmd)
+	if err != nil {
+		return err
+	}
+	spec, err := fileToSpec(flagsImp.file)
+	if err != nil {
+		return err
+	}
+	// Getting Kafka Cluster & SR
+	flagsExport := &flags{
+		kafkaApiKey:             flagsImp.kafkaApiKey,
+		schemaRegistryApiKey:    flagsImp.schemaRegistryApiKey,
+		schemaRegistryApiSecret: flagsImp.schemaRegistryApiSecret,
+	}
+	details, err := c.getAccountDetails(flagsExport)
+	if err != nil {
+		return err
+	}
+	for topicName, topicDetails := range spec.Channels {
+		err := c.addChannelToCluster(details, spec, topicName, topicDetails.Bindings.Kafka, flagsImp.overwrite)
+		if err != nil {
+			if err.Error() == parseErrorMessage {
+				output.Printf("WARNING: \"%s\" %s.\n", topicName, parseErrorMessage)
+			} else {
+				log.CliLogger.Warn(err)
+			}
+		}
+	}
+	return nil
+}
+
+func fileToSpec(fileName string) (*Spec, error) {
+	asyncSpec, err := os.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+	spec := new(Spec)
+	if err := yaml.Unmarshal(asyncSpec, spec); err != nil {
+		return nil, fmt.Errorf("unable to unmarshal YAML file: %v", err)
+	}
+	return spec, nil
+}
+
+func (c *command) addChannelToCluster(details *accountDetails, spec *Spec, topicName string, kafkaBinding kafkaBinding, overwrite bool) error {
+	topicExistedAlready, newTopicCreated, err := c.addTopic(details, topicName, kafkaBinding, overwrite)
+	if err != nil {
+		log.CliLogger.Warn(err)
+	}
+	// If topic exists and overwrite flag is false, move to the next channel in spec
+	if topicExistedAlready && !overwrite {
+		return errors.New(parseErrorMessage)
```

Comment:
Can we put the topic name in this error message, instead of prefixing the warning with `"topic-name":`?